dp83xgui/dpgui.py

```diff
@@ -201,9 +204,9 @@ def __init__(self):
 
         self.channelSpecsDP83x = []
         
-        self.channelSpecsDP83x.append({'DP832A':{'maxV':30,'maxI':3},'DP831':{'maxV':8 ,'maxI':5}})
-        self.channelSpecsDP83x.append({'DP832A':{'maxV':30,'maxI':3},'DP831':{'maxV':30,'maxI':2}})
-        self.channelSpecsDP83x.append({'DP832A':{'maxV':5 ,'maxI':3},'DP831':{'maxV':30,'maxI':2}})
+        self.channelSpecsDP83x.append({'DP832A':{'maxV':30,'maxI':3},'DP832':{'maxV':30,'maxI':3},'DP831':{'maxV':8 ,'maxI':5}})
+        self.channelSpecsDP83x.append({'DP832A':{'maxV':30,'maxI':3},'DP832':{'maxV':30,'maxI':3},'DP831':{'maxV':30,'maxI':2}})
+        self.channelSpecsDP83x.append({'DP832A':{'maxV':5 ,'maxI':3},'DP832':{'maxV':5 ,'maxI':3},'DP831':{'maxV':30,'maxI':2}})
```

Comment:
Also, we could set these limits to higher. On DP832 it's 32V for 1 and 2 and 5.3V for CH3